lib/graphql/schema/field.rb

```diff
@@ -407,8 +407,9 @@ def apply_scope(value, ctx)
       end
 
       # @param ctx [GraphQL::Query::Context::FieldResolutionContext]
+      CONTEXT_EXTRAS = [:path]
```

Comment:
sorry to split hairs, but I think inserting this constant _between_ the method docs and the method signature will break our doc generator. Would you mind moving it up a line?

Reply:
Of course! Consider it done. 😄